game/scripts/npc/abilities/ogre_magi_fireblast.txt

```diff
@@ -27,7 +27,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "11 10 9 8 7 6"
+    "AbilityCooldown"                                     "11 10 9 8 8 8"
```

Comment:
Old multicast reduced cd of this to 4 or something ridiculous fyi,

Reply:
old multicast didnt extend stun duration. Also I think getting past level 30 on supports was unheard of at that time, so you never saw the horror.